packages/next/client/image.tsx

```diff
@@ -310,6 +310,12 @@ export default function Image({
     lazy = false
   }
 
+  if (src.startsWith('data:')) {
```

Comment:
Let's check it here unless we want to make it throw in prod:

```suggestion
  if (src && src.startsWith('data:')) {
```